includes/platform-checkout-user/class-platform-checkout-save-user.php

```diff
@@ -76,4 +83,17 @@ public function maybe_add_userdata_to_metadata( $metadata, $order ) {
 
 		return $metadata;
 	}
+
+	/**
+	 * Clears if platform checkout user data is set.
+	 *
+	 * @return void
+	 */
+	public function maybe_clear_session_key() {
+		$session_data = WC()->session->get( 'platform-checkout-user-data' );
+
+		if ( ! empty( $session_data ) ) {
+			WC()->session->set( 'platform-checkout-user-data', null );
+		}
```

Comment:
```suggestion
		WC()->session->__unset( Platform_Checkout_Extension::PLATMORM_CHECKOUT_SESSION_KEY );
```
Apparently this `__unset` function exists (can see it being used here and there in the codebase): could be used here? Up to you, as I'm not sure which is the better convention.